clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/JsonWebService.scala

```diff
@@ -1,9 +1,22 @@
 package org.broadinstitute.clio.server.webservice
 
 import akka.http.scaladsl.common.{EntityStreamingSupport, JsonEntityStreamingSupport}
+import akka.stream.scaladsl.Flow
+import akka.util.ByteString
 import de.heikoseeberger.akkahttpcirce.ErrorAccumulatingCirceSupport
 import org.broadinstitute.clio.util.json.ModelAutoDerivation
 
+object JsonWebService {
+
+  def singleElememtJsonStreamingSupport: JsonEntityStreamingSupport =
```

Comment:
This could be a `val`.

It also feels weird to have this one explicit, but the other implicit. I guess it's nice to have the other one implicit since multiple endpoints use it...

Reply:
yeah I mean we could be explicit as well.. just seems a bit verbose.. Maybe a comment explaining that one is default and the other should be used when you don't want an array?